docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -0,0 +1,318 @@
+[[conditions-based-autodiscover]]
+= Conditions based autodiscover
+
+You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. You can find more about these inputs and their configurations <<elastic-agent-input-configuration,here>>.
+
+== Example: Target pods by host and label
+
+To automatically identify a Redis Pod and monitor it with the Redis integration, uncomment the following input configuration inside  the {agent} Standalone manifest [here](https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml):
+
+
+[source,yaml]
+------------------------------------------------
+- name: redis
+  type: redis/metrics
+  use_output: default
+  meta:
+    package:
+      name: redis
+      version: 0.3.6
+  data_stream:
+    namespace: default
+  streams:
+    - data_stream:
+        dataset: redis.info
+        type: metrics
+      metricsets:
+        - info
+      hosts:
+        - '${kubernetes.pod.ip}:6379'
+      idle_timeout: 20s
+      maxconn: 10
+      network: tcp
+      period: 10s
+      condition: ${kubernetes.labels.app} == 'redis'
+------------------------------------------------
+
+The condition `${kubernetes.labels.app} == 'redis'` will make the {agent} look for a Pod with the `redis` label app within the scope defined in its manifest. 
+
+List of exported fields of the Kubernetes integration {metricbeat-ref}/exported-fields-kubernetes-processor.html[here] to be used in conditions.
+
+The `redis` input defined in the {agent} manifest only has `info` as metricset. You can find more metricsets in the {metricbeat-ref}/metricbeat-module-redis.html[Redis module page], as well as the configuration options.
```

Comment:
```suggestion
The `redis` input defined in the {agent} manifest only specifies the`info` metricset. To learn about other available metricsets and their configuration settings, refer to the {metricbeat-ref}/metricbeat-module-redis.html[Redis module page].
```
Again I think it will be confusing to users that we point to the module docs, but I don't have a better solution our standalone config docs are incomplete. :-(

Reply:
Indeed that is another good example that we refer to metricsets but we dont explain how agent uses them under the hood. 

@dedemorton does it make sense to give as a note here a small explanation what metricset term is? or it becomes confusing?

Reply:
@dedemorton  https://github.com/elastic/observability-docs/issues/2384 relevant issue to track processors. 
We can follow discussion there but is not urgent

Reply:
@gizas I've commented on your request. I'm actually working on the processors documentation right now.

Reply:
>does it make sense to give as a note here a small explanation what metricset term is? or it becomes confusing?

Actually I think we should be using "dataset" here because that's the correct term to use when talking about Elastic Agent integrations. We are just in a really awkward position right now because the standalone configuration docs are incomplete (a deliberate decision by management to prioritize to Fleet docs over standalone because we have limited resources to work on all the docs).